ZoeKHarvey/Build-Your-Own-Backend
# Start get endpoints

### What's this PR do?
- Adds successful GET endpoints for both albums and songs
- Adds successful GET endpoints for specific albums and specific songs
- Adds successful POST endpoints for both albums and songs
- Returns error code on unsuccessful GET 
- Returns error message with id on unsuccessful POST

### Where should the reviewer start?
Start with the GET method for album/song and then to the GET method for individual albums/songs

How should this be manually tested?
Use given endpoints to test in PostMan

Any background context you want to provide?
Post/Delete were not originally working so I had to download and add body parsing method ( should be good to go now )

What are the relevant tickets?
-User should be able to GET song
-User should be able to GET album
-User should be able to GET specific album
-User should be able to GET specific song
-User should be able to POST a new album
-User should be able to POST a new song
